velox/dwio/common/Options.h

```diff
@@ -491,6 +491,11 @@ class ReaderOptions : public io::ReaderOptions {
     return *this;
   }
 
+  ReaderOptions& setUseParquetColumnNamesForColumnMapping(bool flag) {
```

Comment:
The setter and getters also can be reused, they are format agnostic

Reply:
If we use the same setter, how would the newly introduced parquet config and session properties be set? Or are you suggesting to reuse existing session & config property as [this](https://github.com/facebookincubator/velox/blob/main/velox/connectors/hive/HiveConfig.h#L125) and not add any new property for parquet? Please correct me if I am missing something here.

Reply:
Should be something like this:
```c++
switch (hiveSplit->fileFormat) {
  case dwio::common::FileFormat::DWRF:
  case dwio::common::FileFormat::ORC:
    readerOptions.setUseColumnNamesForColumnMapping(hiveConfig->isOrcUseColumnNames(sessionProperties));
    break;
  case dwio::common::FileFormat::PARQUET:
    readerOptions.setUseColumnNamesForColumnMapping(hiveConfig->isParquetUseColumnNames(sessionProperties));
    break;
};
```